test/Conversion/FIRRTLToRTL/lower-to-rtl.mlir

```diff
@@ -109,13 +109,14 @@ module attributes {firrtl.mainModule = "Simple"} {
     // CHECK-NEXT: = comb.andr [[CONCAT1]] : i8
     %16 = firrtl.andr %6 : (!firrtl.uint<8>) -> !firrtl.uint<1>
 
-    // CHECK-NEXT: = comb.orr [[CONCAT1]] : i8
+    // CHECK-NEXT: = comb.constant(0 : i8) : i8
+    // CHECK-NEXT: = comb.icmp ne {{.*}}, {{.*}} : i8
     %17 = firrtl.orr %6 : (!firrtl.uint<8>) -> !firrtl.uint<1>
 
     // CHECK-NEXT: %c0_i6 = comb.constant
     // CHECK-NEXT: [[ZEXTC1:%.+]] = comb.concat %c0_i6, [[CONCAT1]] : (i6, i8) -> i14
-    // CHECK-NEXT: %c0_i8 = comb.constant
-    // CHECK-NEXT: [[ZEXT2:%.+]] = comb.concat %c0_i8, [[SUB]] : (i8, i6) -> i14
+    // CHECK-NEXT: %c0_i8_6 = comb.constant
+    // CHECK-NEXT: [[ZEXT2:%.+]] = comb.concat %c0_i8_6, [[SUB]] : (i8, i6) -> i14
```

Comment:
Nit: It's annoying that the constant above is causing this to have to unique itself. You can consider going to something like the following, but this may cause other things to bump, too:
```suggestion
    // CHECK-NEXT: [[CONST:%.+]] = comb.constant
    // CHECK-NEXT: [[ZEXT2:%.+]] = comb.concat [[CONST]], [[SUB]] : (i8, i6) -> i14
```